Sources/Workspace/Workspace.swift

```diff
@@ -1119,6 +1120,57 @@ extension Workspace {
             fileSystem: fileSystem,
             observabilityScope: observabilityScope
         )
+
+        try expectedSigningEntities.forEach { identity, expectedSigningEntity in
+            if let package = packageGraph.packages.first(where: { $0.identity == identity }) {
+                if let actualSigningEntity = package.registryMetadata?.signature?.signedBy {
+                    if actualSigningEntity != expectedSigningEntity {
```

Comment:
Should the comparison be more limited?

Reply:
What do you have in mind? e.g., must be `.recognized`?

Reply:
Not sure, just posing the question